### PR TITLE
Add platform simulated

### PR DESCRIPTION
**Pull request by codebynao**
Linked to: https://trello.com/c/Hfw6QLRZ/1208-demo-page-add-other-platforms
platformSimulated will allow us to apply different css based on the platform that should be simulated

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "yelda-webchat",
-  "version": "1.0.50",
+  "version": "1.0.51",
```
Comment (vishnu-ravi) [inline package.json:3]: npm run uploadBuildToS3 command with --semver=patch argument will automatically increment the npm version, so no need to do it manually

Comment (codebynao) [reply package.json:3]: Yes it was from the script actually

**Review by vishnu-ravi — APPROVED** [review]

File: src/js/yeldaWebchatInjector.js [hunk 2]
```diff
@@ -196,6 +196,9 @@ class YeldaChat {
     url = this.updateQueryStringParameter(url, 'assistantSlug', data.assistantSlug)
     url = this.updateQueryStringParameter(url, 'locale', data.locale)
 
+    if (data.platformSimulated) {
+      url = this.updateQueryStringParameter(url, 'platformSimulated', data.platformSimulated)
+    }
     if (data.hasOwnProperty('canBeClosed')) {
       url = this.updateQueryStringParameter(url, 'canBeClosed', data.canBeClosed)
     }
```
Comment (meelie) [inline src/js/yeldaWebchatInjector.js:204]: please add a test 
it should not be very useful, but could be great to keep the habit
image

**Review by meelie — CHANGES_REQUESTED** [review]
ok for the 2 lines changes,
i suggest to add a small unit test for it, but no need to redeploy the plugin for that

**Review by meelie — APPROVED** [review]

**Pull request merged**